lib/phonebook/ui/pages/association_creation_page/text_entry.dart

```diff
@@ -41,8 +41,6 @@ class AddAssociationTextEntry extends StatelessWidget {
                     return AdminTextConstants.emptyFieldError;
```

Comment:
Those two condition are returning the same message, can you just write a single one